### ISSUE TITLE
Weird behavior with fullscreen applications

### ISSUE THREAD

**Issue by Nino-90** [dremin/RetroBar#524]
Hello,
I'm using retrobar for 2 weeks now and I noticed some strange behavior when I'm running some full screen games, like Rocket League for exemple. When I run Rocket League, it goes maximized then immedialety after it switches to desktop, while the game is still running. When I click again on the game in the retrobar it goes maximized then switches to desktop again and again. It runs perfectly when I exit retrobar. That's the most annoying behavior.
I have another minor issue, when I click on an opened Chrome window in retrobar , the Chrome windows shows up but not maximized. The windows height is half his normal size, when I click on the maximize button on Chrome Windows it goes maximized without problem. This issue doesn't happen everytime though.

Thanks for your help

**Comment by dremin** [comment]
Does the issue occur when you switch to the game using alt-tab?

**Comment by Nino-90** [comment]
Thanks for your reply dremin, it happens either with alt tab or when i click on the application on the taskbar.
I hope that it replies to your question.


Another issue, when i go full screen with a VLC clip, the VLC window stay on full screen but the task bar remains visible. 

Thanks for your help

**Comment by NaaeemMalik** [comment]
yeah I am also facing VLC and Chrome bug, VLC bug is persistent and very annoying

**Comment by dremin** [comment]
I believe I have found and fixed a few different ways this could happen. Could you please try [this test build](https://github.com/dremin/RetroBar/actions/runs/13402639334/artifacts/2613032651) and let me know if it works better?

**Comment by Nino-90** [comment]
Hello Dremin, sorry I haven't seen your reply earlier. I recently updated my version to 1.18.56+3691ee4156 and I still have some issues with full screen applications but a lot less than before. Rocket League works now in full screen without any issues, VLC also works perfectly fine. I will test your build as soon as I found an application having difficulties going full screen with Retrobar. Thanks for all your work !

**Comment by dremin** [comment]
Hi @Nino-90, no problem! Here is the latest one with all of the fixes if you'd like to try it: https://github.com/dremin/RetroBar/actions/runs/13752761139/artifacts/2718919650

**Comment by Nino-90** [comment]
Hi @dremin ! I took some time to test my actual version (1.18.56+3691ee4156) and i works perfectly fine. I only had twice the same issue, but it got maximized when I clicked again. By the way, I forgot to mention that I'm on Windows 11. I'm gonna try the version you suggested, and I'll tell you if it still works :)

**Comment by Nino-90** [comment]
Just a little feedback on the build you told me to test, it seems to work a little less well, I have a little more malfunction on full screen applications. I will switch back to latest/stable version. Thanks !
